refinery/factory_boy/django_model_factories.py

```diff
@@ -86,6 +86,7 @@ class GalaxyInstanceFactory(factory.django.DjangoModelFactory):
     """Minimal representation of a GalaxyInstance"""
     class Meta:
         model = "galaxy_connector.Instance"
+    base_url = "www.example.com/galaxy/"
```

Comment:
Curious as to why was this not necessary before e2bb3a6 and also, why is this not necessary when running the tests on Travis?

Reply:
It wasn't necessary before #2385 because there was no logic in place to run `galaxy_cleanup` for  non-successful deleted or cancelled Analyses.

Our GalaxyInstance needs a `base_url` now that we are able to run this cleanup on Analysis deletion.

As for why things weren't failing on Travis I'm not sure.

Reply:
It would be great to find out why test outcome is different in Travis and Vagrant because this could be a symptom of a larger problem. Also, according to the [docs](https://github.com/refinery-platform/refinery-platform/wiki/setting-up-newer-galaxy#configure-refinery-for-your-galaxy-instance), `base_url` should have no trailing slash.